movingpandas/trajectory_generalizer.py

```diff
@@ -203,3 +204,53 @@ def _generalize_traj(self, traj, tolerance):
         new_df = traj.df.iloc[keep_rows]
         new_traj = Trajectory(new_df, traj.id)
         return new_traj
+
+
+class TopDownTimeRatioGeneralizer(TrajectoryGeneralizer):
+    """
+    Generalizes using Top-Down Time Ratio algorithm proposed by Meratnia and De By (DOI: 10.1007/978-3-540-24741-8_44).
+
+<<<<<<< HEAD
+    Points are projected on the line that connects the starting point SP with the end point EP of the trajectory. These projections are
+    calculated based on the ratios de and di that correspond to the travel time from SP to P and from P to EP respectively. Iterativelly,
+    when the furthest point P is found (granted its distance is greater that the tolerance), that point is kept and the process is
+=======
+    Points are projected on the line that connects the starting point (SP) with the end point (EP) of the trajectory. Iterativelly,
+    when the furthest point P is found (granted its distance is greater than the tolerance), that point is kept and the process is
+>>>>>>> 8fd0c34f523eb85566e42765370d830e2634684c
```

Comment:
These two versions have to be merge manually since git couldn't figure it out automatically.

Reply:
Top one is the one we want. Can you do it? I think I will make a mess if try... 

Sorry :)

Reply:
No need I figured it out... :D 💯